icu4c/source/data/brkitr/ja.txt

```diff
@@ -12,14 +12,64 @@ ja{
         line_strict:process(dependency){"line_cj.brk"}
         line_strict_phrase:process(dependency){"line_phrase_cj.brk"}
     }
-    particles{
-         か, かしら, から, が, くらい, けれども, こそ,
-         さ, さえ, しか, だけ, だに, だの, て, で, でも,
-         と, ところが, とも, な, など, なり, に, ね, の,
-         ので, のに, は, ば, ばかり, へ, ほど, まで, も,
-         や, やら, よ, より, わ, を
-    }
-    auxVerbs {
-        です, でしょ, でし, ます, ませ, まし
+    extensions{
+        // Extensions.
+        // Entries in cjdict.txt categorized as particles (助詞), auxiliary verbs (助動詞),
+        // conjugations (活用)  and prefix of them.
+        // These entries are connected to the previous words for phrase based word segmentation.
```

Comment:
@pedberg-icu if we are otherwise in agreement, and you agree with this change as well, then I could commit this change, squash the PR, and merge it. WDYT?

Reply:
@markusicu Yes, I agree with that change, please commit, squash, and merge. Thanks!

Reply:
I committed that typo fix, but I get an error when I try to squash. Apparently I do not have permission to push to @allensu05 's repository. We will need to wait for him to squash.